src/main/java/org/folio/rest/impl/PiecesHelper.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.Context;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.jaxrs.model.Piece;
+import org.folio.rest.tools.client.interfaces.HttpClientInterface;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.orders.utils.ResourcePathResolver.PIECES;
+import static org.folio.orders.utils.ResourcePathResolver.resourcesPath;
+
+public class PiecesHelper extends AbstractHelper {
```

Comment:
I thought you was going to move pieces related methods from `PurchaseOrderLineHelper` to this helper as well as part of this PR. But it is okay to do it later...